src/application-manager.coffee

```diff
@@ -862,11 +862,10 @@ module.exports = class ApplicationManager extends EventEmitter
 			if !localMode
 				currentAppIds = _.keys(current.local.apps).map((n) -> checkInt(n))
 				targetAppIds = _.keys(target.local.apps).map((n) -> checkInt(n))
-				if targetAppIds.length > 1
-					throw new Error('Current supervisor does not support multiple applications')
+
 				diff = _.difference(currentAppIds, targetAppIds)
 				if diff.length > 0
-					oldApps = diff
+					appsForVolumeRemoval = diff
```

Comment:
NIT: this could be `appsForVolumeRemoval = _.difference(currentAppIds, targetAppIds)` and the check line line 901 could become `if _.includes(appsForVolumeRemoval, checkInt(appId))`